spec/rubocop/cop/style/method_call_with_args_parentheses_spec.rb

```diff
@@ -1327,4 +1327,81 @@ module Foo
       end
     end
   end
+
+  context 'when inspecting macro methods with IncludedMacroPatterns' do
+    let(:cop_config) do
+      { 'IgnoreMacros' => 'true', 'IncludedMacroPatterns' => ['^assert', '^refute'] }
+    end
+
+    it_behaves_like 'endless methods'
+
+    context 'in a class body' do
+      it 'finds offense for methods matching patterns' do
+        expect_offense(<<~RUBY)
+          class Foo
+            assert_equal 'expected', actual
+            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use parentheses for method calls with arguments.
+            refute_nil value
+            ^^^^^^^^^^^^^^^^ Use parentheses for method calls with arguments.
+          end
+        RUBY
+
+        expect_correction(<<~RUBY)
+          class Foo
+            assert_equal('expected', actual)
+            refute_nil(value)
+          end
+        RUBY
+      end
+    end
+
+    context 'for macros called within a block' do
+      it 'finds offense' do
+        expect_offense(<<~RUBY)
+          class Foo
+            test "description" do
+              assert_equal 1, 1
+              ^^^^^^^^^^^^^^^^^ Use parentheses for method calls with arguments.
+            end
+          end
+        RUBY
```

Comment:
Could you add a test like this to the `'when inspecting macro methods with IncludedMacros'` context for when `assert_equal` is within a `test` block but configured in `IncludedMacros`? While I understand why you want to add a configuration for patterns, the old behaviour should still be acknowledged.